File: UserInterface/ClientApp/src/app/model/weather-condition.model.ts
import { Time } from "@angular/common";



export class WeatherCondition {
    timezone: number;
    visibility: string;
    dewPoint: string;
    coord: Coord;
    time: Date;
    weather: Array<Weather>;
    main: Main;

}

export class Coord {
    lon: string;
    lat: string;
}

export class Weather {
    id: string;
    main: string;
    description: string;
}

export class Main {
    temp: string;
    tempC: string;
    pressure: string;
    humidity: string;
}

export class Wind {
    speed: string;
    deg: string;
}

File: Models/WeatherConditionModel.cs
using Models.WeatherConditionDetail;
using System;
using System.Collections.Generic;

namespace Models
{
    public class WeatherConditionModel
    {
        public CoordModel Coord { get; set; }
        public List<WeatherModel> Weather { get; set; }
        public MainModel Main { get; set; }
        public WindModel Wind { get; set; }
        public DateTime Time { get; set; }
        public int Timezone { get; set; }
        public string Visibility { get; set; }
        public string DewPoint { get; set; }
    }
}
